spec/features/add_an_ip_address_spec.rb

```diff
@@ -28,8 +28,8 @@
       expect { click_on 'save' }.to change { Ip.count }.by 1
 
       expect(Ip.first.user).to eq(user)
-      expect(page).to have_content('New IP Added!')
-      expect(page).to have_content(Ip.last.address)
+      expect(page).to have_content('IP Added')
+      expect(page).to have_content(user.radius_secret_key)
```

Comment:
We could keep `expect(page).to have_content(Ip.last.address)` as well? Or was there a problem with it?